database/resources/needs.ts

```diff
@@ -1,6 +1,9 @@
 import { Need } from 'models'
 import { ResourcesAirtableClient } from '.'
+import { FieldSet } from 'airtable';
 
-export const selectAllNeeds = (filterFormula?: string): Promise<Need[]> => {
-  return ResourcesAirtableClient.selectAll<Need>('Ref - Need', filterFormula)
+export const selectAllNeeds = (filterFormula?: string): Promise<FieldSet[]> => {
+  // TODO: Figure out how to get all needFocus options or create a needs focus table
```

Comment:
These value options are from the`taxonomy_terms` table, but only the ones with "MANYC Need" as the value in the "taxonomy" field. Is it possible to use that?